bzira.py

```diff
@@ -211,7 +215,7 @@ def parse_options():
 
     if(options.verbose):
         print '[DEBUG] %s - %s [%s, %s, [%s]] {%s} -> %s (%s)' % (bug.id, bug.summary, bug.product, bug.component, bug.target_milestone, bug.delta_ts, bug.weburl, difference)
-    else:
+    elif(options.dryrun is None): 
```

Comment:
Yes, but in dry run mode you get a whole line and the . right before the [INFO] looks silly